WordPress/gutenberg
# Deprecated the selectors/actions and components that moved to the block editor module

**Testing instructions**

- Check that if you call, `wp.data.select( 'core/editor' ).getSelectedBlock()` in the console it works but shows a warning.

- Change an `import RichText` and use `editor` instead of `block-editor` in one of the core blocks and check that there are warnings on the console when used.
